src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1414,7 +1414,9 @@ uct_ib_mlx5_devx_reg_exported_key(uct_ib_md_t *ib_md, uct_ib_mem_t *ib_memh)
     umem_in.addr        = address;
     umem_in.size        = length;
     umem_in.access      = UCT_IB_MLX5_MD_UMEM_ACCESS;
-    umem_in.pgsz_bitmap = UINT64_MAX & ~UCS_MASK(UCT_IB_MLX5_PAGE_SHIFT);
+    /* TODO: Allow other page sizes as well when umem offset handling is
+     * fixed in the driver. */
+    umem_in.pgsz_bitmap = ucs_get_page_size();
```

Comment:
AFAIR (need to check with Jason) since the driver doesn't know UMEM will be used for MKEY, it leaves possible page sizes selection to the application.
Maybe we could pass `UCS_MASK(ucs_ffs64(address))` so the driver still will be able to pick a larger page size aligned with `address`?